patches/minecraft/net/minecraft/world/level/storage/loot/LootTables.java.patch

```diff
@@ -6,7 +6,7 @@
           try {
 -            LootTable loottable = f_79189_.fromJson(p_79199_, LootTable.class);
 +            net.minecraft.server.packs.resources.Resource res = p_79215_.m_213713_(getPreparedPath(p_79198_)).orElse(null);
-+            LootTable loottable = net.minecraftforge.common.ForgeHooks.loadLootTable(f_79189_, p_79198_, p_79199_, res == null || !res.m_215506_().equals("Default"), this);
++            LootTable loottable = net.minecraftforge.common.ForgeHooks.loadLootTable(f_79189_, p_79198_, p_79199_, res == null || !res.m_215506_().equals("vanilla"), this);
```

Comment:
This should use the constant `BuiltInPackSource#VANILLA_ID` instead of a hardcoded string.